packages/fxa-admin-server/src/scripts/audit-tokens.ts

```diff
@@ -478,8 +479,20 @@ export async function run() {
 if (require.main === module) {
   process.on('exit', (code) => log.info('exit', { code }));
 
+  const checkInId = Sentry.captureCheckIn({
```

Comment:
Before capturing, we need initialize sentry in this process and configure a monitor. Check the docs [here](https://docs.sentry.io/platforms/python/guides/logging/crons/#requirements).

We should double check with SRE, but I think the process's SENTRY_DSN environment variable, will already be populated. If so, all we need to do is `Sentry.Init({});` once at startup and it should fallback back to the `SENTRY_*` environment variables. If not, it's easy enough to include these envs in our deployment config.